dbt/adapters/bigquery/column.py

```diff
@@ -13,12 +13,9 @@
 @dataclass(init=False)
 class BigQueryColumn(Column):
     TYPE_LABELS = {
-        "STRING": "STRING",
```

Comment:
Shouldn't we just be adding `TEXT`? Why delete the other types?

Reply:
There is no point to types that are the same on both sides. They do nothing except (in some cases) change the type from lowercase to uppercase.

Reply:
I removed them in the dbt-core PR because they were causing unnecessary churning in the test cases.